plugin/core/src/test/resources/ColorizerTest/colorize.json

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
Wow, I'd almost forgotten about this. I believe I based it on the 9patch json stuff, so:

```javascript
{
  matches: [ /* list of filename patterns the change applies to */ ]
, colors: [
    {
      from: // old color (to be replaced)
    , to: // new color
    }
  ]
}
```

Like `ninePatchConfig`, the file is selected by passing a file path to `colorizerConfig` in build.gradle